portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fun fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const funFacts =
+      ["I've built 8 computers so far!", 'I interned at Google last summer.', 'I spent my last summer in New York City.', 'I speak Spanish.'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const funFact = funFacts[Math.floor(Math.random() * funFacts.length)];
```

Comment:
This sometimes causes you to randomly get the same fact twice in a row which feels like a broken experience if I click the button twice and the fact doesn't change. Do you think we can make it so it'll never pick the same one twice in a row?

Reply:
Done.